### PR TITLE
Feature #121004 feat: widgets and menus services using loopback node …

### PR DESCRIPTION
**Pull request by ranjit-redekar**
…module

### CODE REVIEW

File: server/common/models/widget.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "widget",
+  "plural": "widgets",
+  "base": "PersistedModel",
+  "idInjection": true,
+  "options": {
+    "validateUpsert": true
+  },
+  "properties": {
+    "widName": {
```
Comment (coolbung) [inline server/common/models/widget.json:10]: Lets use a consistent naming convention. Please update all code to use a consistent naming.
- Decide if you want to use short forms or full words
- Decide between camel case or underscore

File: web/src/app/unite/services/menus.service.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+@Injectable()
+export class MenusService extends BaseService{
+
+  constructor(_httpClient: HttpClient) {
+    super(_httpClient);
+  }
+
+  getMenus() {
+    return this.get('http://172.132.45.94:3000/api/Menus');
```
Comment (sushan05) [inline web/src/app/unite/services/menus.service.ts:13]: @ranjit-redekar - get the base path for API from environment variables

Comment (ranjit-redekar) [reply web/src/app/unite/services/menus.service.ts:13]: Ok I will update this in new MR, I have pushed this MR for only for backup.

Comment (sushan05) [reply web/src/app/unite/services/menus.service.ts:13]: @ranjit-redekar - ok

**Pull request merged**